lxd/storage/drivers/utils_cgo.go

```diff
@@ -348,3 +360,25 @@ func UnsetAutoclearOnLoopDev(loopFd int) error {
 
 	return nil
 }
+
+func SetDirectIOLoopDev(loopFd int) error {
+	ret, err := C.set_direct_io_loop_device(C.int(loopFd))
+	if ret < 0 {
+		if err != nil {
+			return err
+		}
+		return fmt.Errorf("Failed to set LO_FLAGS_DIRECT_IO")
+	}
+	return nil
+}
+
+func UnsetDirectIOLoopDev(loopFd int) error {
+	ret, err := C.unset_direct_io_loop_device(C.int(loopFd))
+	if ret < 0 {
+		if err != nil {
+			return err
+		}
+		return fmt.Errorf("Failed to unset LO_FLAGS_DIRECT_IO")
+	}
+	return nil
+}
```

Comment:
Those two aren't used anywhere, so they should be safe to drop (and the cgo code along with them).